device.h

```diff
@@ -11,6 +11,9 @@
 #define PIXFMTS_MAX 4
 #define FB_NAME_MAXLENGTH 16
 
+#define LD_360_WIDTH 480
+#define LD_360_HEIGHT 360
```

Comment:
Drop the above definitions (480, 360) since the were Intendedly used once. Move it to file `device.c` instead.